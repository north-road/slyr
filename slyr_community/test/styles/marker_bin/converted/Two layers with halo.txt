[{'class': 'QgsSimpleMarkerSymbolLayer',
  'offset': '17,-19',
  'offset_unit': 'Point',
  'outline_style': 'no',
  'size': '15',
  'size_unit': 'Point'},
 {'class': 'QgsSimpleMarkerSymbolLayer',
  'color': '0,0,255,255',
  'offset': '11,-13',
  'offset_unit': 'Point',
  'outline_style': 'no',
  'size': '9',
  'size_unit': 'Point'}]
