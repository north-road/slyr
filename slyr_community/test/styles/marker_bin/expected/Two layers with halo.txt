{'halo': True,
 'halo_size': 14.0,
 'halo_symbol': {'levels': [{'color': {'B': 0,
                                       'G': 255,
                                       'R': 0,
                                       'dither': False,
                                       'is_null': False,
                                       'stream_offset': '0x129',
                                       'type': 'RgbColor',
                                       'version': 1},
                             'enabled': True,
                             'fill_style': 'solid',
                             'locked': False,
                             'outline': {'color': {'B': 240,
                                                   'G': 240,
                                                   'R': 240,
                                                   'dither': True,
                                                   'is_null': True,
                                                   'stream_offset': '0xe6',
                                                   'type': 'RgbColor',
                                                   'version': 1},
                                         'enabled': True,
                                         'line_type': 'solid',
                                         'locked': False,
                                         'stream_offset': '0xd4',
                                         'symbol_level': 0,
                                         'type': 'SimpleLineSymbol',
                                         'version': 1,
                                         'width': 1.0},
                             'stream_offset': '0xc2',
                             'symbol_level': 0,
                             'type': 'SimpleFillSymbol',
                             'version': 1}],
                 'stream_offset': '0x75',
                 'symbol_level': 0,
                 'type': 'MultiLayerFillSymbol',
                 'version': 2},
 'levels': [{'angle': 0.0,
             'color': {'B': 0,
                       'G': 0,
                       'R': 255,
                       'dither': False,
                       'is_null': False,
                       'stream_offset': '0x188',
                       'type': 'RgbColor',
                       'version': 1},
             'enabled': True,
             'locked': False,
             'marker_type': 'circle',
             'outline_color': {'B': 0,
                               'G': 0,
                               'R': 0,
                               'dither': False,
                               'is_null': False,
                               'stream_offset': '0x1ec',
                               'type': 'RgbColor',
                               'version': 1},
             'outline_enabled': False,
             'outline_size': 1.0,
             'rotate_with_transform': True,
             'size': 15.0,
             'stream_offset': '0x176',
             'symbol_level': 0,
             'type': 'SimpleMarkerSymbol',
             'version': 2,
             'x_offset': 17.0,
             'y_offset': 19.0},
            {'angle': 0.0,
             'color': {'B': 255,
                       'G': 0,
                       'R': 0,
                       'dither': True,
                       'is_null': False,
                       'stream_offset': '0x22f',
                       'type': 'RgbColor',
                       'version': 1},
             'enabled': True,
             'locked': True,
             'marker_type': 'circle',
             'outline_color': {'B': 0,
                               'G': 0,
                               'R': 0,
                               'dither': False,
                               'is_null': False,
                               'stream_offset': '0x293',
                               'type': 'RgbColor',
                               'version': 1},
             'outline_enabled': False,
             'outline_size': 1.0,
             'rotate_with_transform': True,
             'size': 9.0,
             'stream_offset': '0x21d',
             'symbol_level': 0,
             'type': 'SimpleMarkerSymbol',
             'version': 2,
             'x_offset': 11.0,
             'y_offset': 13.0}],
 'stream_offset': '0x0',
 'symbol_level': 0,
 'type': 'MultiLayerMarkerSymbol',
 'version': 3}
